doc/source/whatsnew/v2.3.0.rst

```diff
@@ -127,7 +127,7 @@ MultiIndex
 
 I/O
 ^^^
--
+- ``pyarrow`` engine for :func:`read_parquet` accepts ``to_pandas_kwargs`` which are forwarded to :meth:`pyarrow.Table.to_pandas`. This enables passing in ``maps_as_pydicts`` to read parquet map datatypes as python dictionaries. (:issue:`56842`)
```

Comment:
Can you move this to the 3.0.0.rst file? At the moment general new features are targeted for 3.0.0

Reply:
done